walkthough_steps.md

```diff
@@ -1,7 +1,7 @@
 # Walkthrough steps
 
 As shown at the beginning of the video, you can select what you want to include in your video and in what order you want to cover them.
-When recording as you jump to the next step you will automatically be taken to the exact page or section for that element.
+When recording as you jump to the next step you will automatically be taken to the exact page or section for that element (where possible).
```

Comment:
Is the `where possible` note only for?

Reply:
> Is the `where possible` note only for?

I've recorded a video reply:

[9s - 15s](http://localhost:8000/video/video-code-review/video-walkthrough/1/eyJ1c2VyIjoicHllcHllIiwibnVtYmVyIjoxfQ.Go2yV3jw_tyxS2ur5U0g2FCZtoc/?start=9&end=15): Is the `where possible` note only for?

Reply:
> Is the `where possible` note only for?

I've recorded a video reply:

[138s - 149s](http://localhost:8000/video/video-code-review/video-walkthrough/1/eyJ1c2VyIjoicHllcHllIiwibnVtYmVyIjo0fQ.TJPC78dsBS_HEuBmyhoeW8Vdb-8/?start=138&end=149): Is the `where possible` note only for?

Reply:
> Is the `where possible` note only for?

I've recorded a video reply:

[00:10 - 00:10](http://localhost:8000/video/video-code-review/video-walkthrough/1/eyJ1c2VyIjoicHllcHllIiwibnVtYmVyIjo1fQ.aH60pVvIgGSBHlk4xXffHcEUtmk/?start=70&end=90): Is the `where possible` note only for?

Reply:
> Is the `where possible` note only for?

I've recorded a video reply:

[00:54 - 01:16](http://localhost:8000/video/video-code-review/video-walkthrough/1/eyJ1c2VyIjoicHllcHllIiwibnVtYmVyIjo2fQ.d_No1HMSSGw6FMeUe8QCfoMzC3Y/?start=54&end=76): Is the `where possible` note only for?

Reply:
> Is the `where possible` note only for?

I've recorded a video reply:

[00:12 - 00:26](http://localhost:8000/video/video-code-review/video-walkthrough/1/WyJweWVweWUiLCJtcDQiLDE2MTI1NzYwMzFd.4sPoSuBHMjkXo4ISeT72YqvJMN0/?start=12&end=26): Is the `where possible` note only for?

Reply:
> Is the `where possible` note only for?

I've recorded a video reply:

[00:12 - 00:23](http://localhost:8000/video/video-code-review/video-walkthrough/1/WyJweWVweWUiLCJtcDQiLDE2MTI1NzY0MTFd.q3k_zZYcBrevf_-mRR2EmkYYGaQ/?start=12&end=23): Is the `where possible` note only for?

Reply:
> Is the `where possible` note only for?

I've recorded a video reply:

[00:19 - 00:22](http://localhost:8000/video/video-code-review/video-walkthrough/1/WyJweWVweWUiLCJtcDQiLDE2MTI4OTUzMjRd.gTdHKK37VBTnQtVMaVp_I8WQIK4/?start=19&end=22): Is the `where possible` note only for?